pkg/k8shandler/kibana/reconciler.go

```diff
@@ -356,6 +355,11 @@ func isDeploymentDifferent(current *apps.Deployment, desired *apps.Deployment) (
 		different = true
 	}
 
+	if !utils.EnvValueEqual(current.Spec.Template.Spec.Containers[1].Env, desired.Spec.Template.Spec.Containers[1].Env) {
```

Comment:
How solid is the assumption that index 1 corresponds to the proxy container?

Reply:
very high since we always add it in as the second one. I've never seen k8s change that order once pushed....

Reply:
May be loop over all the containers then?

Reply:
good point, we do this already for the images

Reply:
@syedriko addressed